src/client/src/styleguide/index.tsx

```diff
@@ -45,6 +45,8 @@ const Styleguide: FC = () => {
       </Helmet>
       <Root>
         <FloatingTools />
+        {/* eslint-disable-next-line @typescript-eslint/ban-ts-comment */}
```

Comment:
ack, this is unfortunate .. anything we can do in future to avoid these messy commented exclusions?

Reply:
yes, we can fix the underlying typing issues, but there are a number of them in the styleguide so the effort would be non-trivial, and given the styleguide is not part of the actual application, I made the decision to not address all of them as part of this effort.